### PR TITLE
Increase dragging threshold

### LINKED ISSUE #5155: Tiling drag: Accidentally clicking into the title has a high chance to move your window in an unwanted split.
## I'm submitting a…
<pre>
[x] Bug
[ ] Feature Request
[ ] Documentation Request
[ ] Other (Please describe in detail)
</pre>

## Current Behavior
Accidentally clicking into the title has a high chance to move your window in an unwanted split.

## Expected Behavior
The user can opt-out of this feature and reclaim safe title mouse interactions.

## Reproduction Instructions
Click around in your title, even minor mouse moves during the press&release events will cause the window to split.

## Environment
Output of `i3 --moreversion 2>&-`:
<pre>
Binary i3 version:  4.21 (2022-09-21) © 2009 Michael Stapelberg and contributors
Running i3 version: 4.21 (2022-09-21) (pid 1056)
</pre>

<pre>
- Are you using a compositor (e.g., xcompmgr or compton): No
</pre>

### PR DESCRIPTION
**Pull request by orestisfl**
Just new random magic number. I didn't notice anything obviously broken with this patch.

Fixes #5155

### CODE REVIEW

**Comment by orestisfl** [comment]
As the tests show, $mod+drag should probably never be affected by the threshold.

File: src/drag.c [hunk 1]
```diff
@@ -42,7 +42,7 @@ struct drag_x11_cb {
 
 static bool threshold_exceeded(uint32_t x1, uint32_t y1,
                                uint32_t x2, uint32_t y2) {
-    const uint32_t threshold = 9;
+    const uint32_t threshold = 50;
```
Comment (stapelberg) [inline src/drag.c:45]: This value should be run through `logical_px()`, right?

**Review by stapelberg — COMMENTED** [review]
Do you want to include the config option in this PR as well, or should that be separate?

**Comment by stapelberg** [comment]
Let’s move forward with https://github.com/i3/i3/pull/5208 instead.

**Pull request closed without merging**